projects/js-packages/licensing/components/activation-screen-controls/error-gridicon.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const ErrorGridicon = () => {
```

Comment:
Would it be an option to import the icon from `@wordpress/icons` instead?

Reply:
`warning` from `@wordpress/icons` ( the bottom icon ) seems like an acceptable substitute to the original ( the top icon )
<img width="125" alt="Screen Shot 2021-11-15 at 10 14 10 AM" src="https://user-images.githubusercontent.com/2810519/141833223-9a173b83-5483-45d7-9cec-0fe4014395ba.png">